slack-plugin/src/main/kotlin/slack/gradle/ApkVersioningPlugin.kt

```diff
@@ -98,14 +113,28 @@ internal class ApkVersioningPlugin : Plugin<Project> {
 
   private fun configureVariants(
     project: Project,
-    versionNameProvider: Provider<String>,
-    versionCodeProvider: Provider<Int>
+    debugVersionNameProvider: Provider<String>,
+    debugVersionCodeProvider: Provider<Int>,
+    defaultVersionNameProvider: Provider<String>,
+    defaultVersionCodeProvider: Provider<Int>,
   ) {
     // Register version calculations for variants
     project.configure<ApplicationAndroidComponentsExtension> {
       onVariants { variant ->
         val flavorName = variant.flavorName
 
+        val versionCodeProvider =
+          if (variant.buildType == "debug") {
+            debugVersionCodeProvider
+          } else {
+            defaultVersionCodeProvider
+          }
```

Comment:
Does this mean debug CI build will have `90009999` number from now?

Reply:
yep